components/director/internal/domain/scenarioassignment/resolver.go

```diff
@@ -67,3 +68,32 @@ func (r *Resolver) SetAutomaticScenarioAssignment(ctx context.Context, in graphq
 
 	return &gqlApp, nil
 }
+
+func (r *Resolver) GetAutomaticScenarioAssignmentByScenarioName(ctx context.Context, scenarioName string) (*graphql.AutomaticScenarioAssignment, error) {
+	tx, err := r.transact.Begin()
+	if err != nil {
+		return nil, errors.Wrap(err, "while beginning transaction")
+	}
+	defer r.transact.RollbackUnlessCommited(tx)
+
+	ctx = persistence.SaveToContext(ctx, tx)
+
+	tnt, err := tenant.LoadFromContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	out, err := r.svc.GetByScenarioName(ctx, tnt, scenarioName)
+	if err != nil {
+		return nil, errors.Wrap(err, "while receiving Assignment")
```

Comment:
Receiving is a bit misleading in this context
```suggestion
		return nil, errors.Wrap(err, "while getting Assignment")
```